src/services/fileStorage/strategies/awsS3.js

```diff
@@ -80,15 +81,15 @@ const FEATURE_MULTIPLE_S3_PROVIDERS_ENABLED = Configuration.get('FEATURE_MULTIPL
 // begin legacy
 let awsConfig = {};
 if (!FEATURE_MULTIPLE_S3_PROVIDERS_ENABLED) {
-	try {
-		//	awsConfig = require(`../../../../config/secrets.${prodMode ? 'js' : 'json'}`).aws;
-		/* eslint-disable global-require, no-unused-expressions */
-		(['production'].includes(process.env.NODE_ENV))
-			? awsConfig = require('../../../../config/secrets.js').aws
-			: awsConfig = require('../../../../config/secrets.json').aws;
-		/* eslint-enable global-require, no-unused-expressions */
-	} catch (e) {
-		logger.warning('The AWS config couldn\'t be read');
+try {
+	//	awsConfig = require(`../../../../config/secrets.${prodMode ? 'js' : 'json'}`).aws;
+	/* eslint-disable global-require, no-unused-expressions */
+	(NODE_ENV === ENVIRONMENTS.PRODUCTION)
+		? awsConfig = require('../../../../config/secrets.js').aws
+		: awsConfig = require('../../../../config/secrets.json').aws;
+	/* eslint-enable global-require, no-unused-expressions */
+} catch (e) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 1 tab but found 0.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5246313)